schema/elasticsearch/visibility/versioned/v2/reindex.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Let's give it more generic name. `upgrade.sh` ?

Reply:
sounds good